java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -67,6 +66,8 @@
 @Rule(key = "S2095")
 public class UnclosedResourcesCheck extends SECheck {
 
+  private static final ArrayList<Class<? extends Constraint>> RESOURCE_CONSTRAINT_DOMAIN = Lists.newArrayList(ResourceConstraint.class);
```

Comment:
`ArryList` ? Isn't `List` working as well?

Reply:
srry, this was stupid refactoring done too fast